releasenotes-builder/src/test/java/com/github/checkstyle/internal/AbstractPathTestSupport.java

```diff
@@ -42,6 +42,23 @@ public abstract class AbstractPathTestSupport {
     protected static final String MISC = Constants.MISCELLANEOUS_LABEL;
     protected static final String BREAKING = Constants.BREAKING_COMPATIBILITY_LABEL;
 
+    protected static final String MSG_EXECUTION_SUCCEEDED = System.lineSeparator()
+            + "Execution succeeded!" + System.lineSeparator();
+
+    protected static final String MSG_RELEASE_IS_MINOR =
+        "\nGeneration ends with 1 errors.\n\n[ERROR] Validation of release number failed. "
```

Comment:
I see usages of `\n` and `System.lineSeparator()`. We should choose one or the other. Beware that if we use JVM's default line separator in production code, Windows uses `\r\n` instead of `\n`.

Reply:
https://ci.appveyor.com/project/Checkstyle/contribution/builds/45632831/job/h7uoii387gp7gu7c#L91
CI agrees.

Reply:
Wasn't aware of that and was wondering why tests failed there. Thanks! Done.